src/commands/build/wranglerjs/mod.rs

```diff
@@ -67,22 +139,17 @@ pub fn run_build(
         command.arg(format!("--use-entry={}", package_main));
     }
 
-    info!("Running {:?}", command);
-
-    let status = command.status()?;
-    let output = fs::read_to_string(temp_file.clone()).expect("could not retrieve ouput");
-    fs::remove_file(temp_file)?;
-
-    if status.success() {
-        Ok(serde_json::from_str(&output).expect("could not parse wranglerjs output"))
-    } else {
-        failure::bail!("failed to execute `{:?}`: exited with {}", command, status)
+    if watch {
+        command.arg("--watch=1");
```

Comment:
we could pass `command.arg("--watch={:?}", watch)`

Reply:
done